src/components/forms/SubnetTierForm.js

```diff
@@ -151,6 +164,25 @@ class SubnetTierForm extends React.Component {
             this.setState({ showUnsavedChangesModal: false });
           }}
         />
+        <IcseModal
+          id={this.props.data.name + "-avanced-save"}
+          name="Enable Advanced Configuration"
+          heading={this.props.data.name}
```

Comment:
should we make the heading `Enable Advanced Configuration`? I don't think setting the heading of the modal as the tier name is very descriptive for why the modal is popping up

Reply:
sounds good